firmware/applications/SignBuddy/sensors.c

```diff
@@ -22,12 +23,14 @@
 #define TIMEOUT_MS           40
 #define TIMEOUT_TICKS        pdMS_TO_TICKS(TIMEOUT_MS)
 
-#define SAMPLING_PERIOD      1000
+#define SAMPLING_PERIOD      50
+#define MAX_SAMPLES          40
 
 typedef struct {
   TaskHandle_t       sensors_task_handle;
   EventGroupHandle_t data_ready_event_group;
   TimerHandle_t      sampling_timer;
+  Sample             sample;
```

Comment:
this type name is not great - is it defined in the proto?

Reply:
Yeah it's defined in the proto